src/components/MakeWidget/MakeWidget.tsx

```diff
@@ -445,14 +445,7 @@ const MakeWidget: FC = () => {
               )}
             </TooltipContainer>
           ) : (
-            <StyledInputSection onInfoButtonClick={toggleShowOrderTypeInfo}>
```

Comment:
Do we still want to disable this?

Reply:
We need community storage to be hardened a bit before enabling.